web/beacon-app/src/components/layout/Sidebar/Sidebar.tsx

```diff
@@ -72,6 +81,37 @@ function SideBar({ className }: SidebarProps) {
     invariant(appState?.orgID, 'orgID is not defined');
   }, [appState?.orgID]);
 
+  // Get the version number from the status response without the release level and number.
+  const statusVersion = status?.version.match(/^v?(\d+\.\d+\.\d+)/);
+
+  useEffect(() => {
+    const storedAppVersion = localStorage.getItem('appVersion');
+
+    if (env !== 'development') {
+      if (storedAppVersion && storedAppVersion !== statusVersion?.[0]) {
+        const updateAlertToast = toast.error(
+          <div className="flex items-center">
+            <UpdateAlert />
+          </div>,
+          {
+            // Display toast until the user clicks the update button.
+            duration: 999999999,
+            position: 'bottom-right',
+            icon: '🔔',
+            style: {
+              background: '#EBF5FF',
+              color: '#1E429F',
+            },
+          }
+        );
+
+        return () => {
+          toast.dismiss(updateAlertToast);
```

Comment:
This ensures that only this toast message is dismissed after a user clicks the `Update` button. If another toast appears, then it shouldn't be dismissed.